readme.md

```diff
@@ -26,36 +26,45 @@ Technical Language Processing (TLP) is a set of tools, techniques, and guideline
 
 ## Contents
 
+- [What is TLP](#what-is-tlp)
 - [TLP Tools](#tlp-tools)
 - [TLP Datasets](#tlp-datasets)
-- [TLP Papers](#tlp-papers)
 - [TLP Resources](#tlp-resources)
+- [Human Centric TLP Research](#tlp-human)
 - [Follow](#follow)
 
+### Legend: 📃 paper - 🖥️ software tool - 🗄️ dataset - 🏷️ model - 📘 standard - 🔌 library 
+
 <!-- CONTENT -->
 
-## TLP Tools
+## What is TLP?  
```

Comment:
```suggestion
## What is TLP
```
Can't have punctuation right now, is have to figure out how to make an exception